react-setup/src/pages/Popup/popup.js

```diff
@@ -0,0 +1,5 @@
+function AddAlarm_click(course_no,course_details, dates){
+  console.log(course_no, course_details,  dates);
+}
+
+export { AddAlarm_click }
```

Comment:
Why this file is created?

Reply:
this is to give a refrence of function to darshit to create further code in the file to add alarms

Reply:
remove it from pr

Reply:
removed!